src/grpc_interop.cc

```diff
@@ -36,66 +36,6 @@ namespace grpc_labview
         DSDisposeHandle(eventData.methodName);
     }
 
-    //---------------------------------------------------------------------
```

Comment:
Note: This code as moved into the constructor of the `LVMessageMetadata` class itself as that seemed more appropriate division of responsibility.